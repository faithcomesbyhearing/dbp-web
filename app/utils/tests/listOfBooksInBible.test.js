import booksList from '../listOfBooksInBible';

describe('List of books in bible utility data', () => {
	it('should match previous snapshot', () => {
		expect(booksList).toMatchSnapshot();
	});
});
