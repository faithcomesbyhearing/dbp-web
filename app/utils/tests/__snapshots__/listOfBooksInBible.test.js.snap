// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List of books in bible utility data should match previous snapshot 1`] = `
{
  "1CH": "1 Chronicles",
  "1CO": "1 Corinthians",
  "1JN": "1 John",
  "1KI": "1 Kings",
  "1PE": "1 Peter",
  "1SA": "1 Samuel",
  "1TH": "1 Thessalonians",
  "1TI": "1 Timothy",
  "2CH": "2 Chronicles",
  "2CO": "2 Corinthians",
  "2JN": "2 John",
  "2KI": "2 Kings",
  "2PE": "2 Peter",
  "2SA": "2 Samuel",
  "2TH": "2 Thessalonians",
  "2TI": "2 Timothy",
  "3JN": "3 John",
  "ACT": "Acts",
  "AMO": "Amos",
  "COL": "Colossians",
  "DAN": "Daniel",
  "DEU": "Deuteronomy",
  "ECC": "Ecclesiastes",
  "EPH": "Ephesians",
  "EST": "Esther",
  "EXO": "Exodus",
  "EZK": "Ezekiel",
  "EZR": "Ezra",
  "GAL": "Galatians",
  "GEN": "Genesis",
  "HAB": "Habakkuk",
  "HAG": "Haggai",
  "HEB": "Hebrews",
  "HOS": "Hosea",
  "ISA": "Isaiah",
  "JAS": "James",
  "JDG": "Judges",
  "JER": "Jeremiah",
  "JHN": "John",
  "JOB": "Job",
  "JOL": "Joel",
  "JON": "Jonah",
  "JOS": "Joshua",
  "JUD": "Jude",
  "LAM": "Lamentations",
  "LEV": "Leviticus",
  "LUK": "Luke",
  "MAL": "Malachi",
  "MAT": "Matthew",
  "MIC": "Micah",
  "MRK": "Mark",
  "NAM": "Nahum",
  "NEH": "Nehemiah",
  "NUM": "Numbers",
  "OBA": "Obadiah",
  "PHM": "Philemon",
  "PHP": "Philippians",
  "PRO": "Proverbs",
  "PSA": "Psalm",
  "REV": "Revelation",
  "ROM": "Romans",
  "RUT": "Ruth",
  "SNG": "Song of Songs",
  "TIT": "Titus",
  "ZEC": "Zechariah",
  "ZEP": "Zephaniah",
}
`;
